public/app/plugins/datasource/influxdb/components/VisualInfluxQLEditor/AddButton.tsx

```diff
@@ -16,6 +16,9 @@ export const AddButton = ({ loadOptions, allowCustomValue, onAdd }: Props): JSX.
       loadOptions={loadOptions}
       allowCustomValue={allowCustomValue}
       onChange={(v) => {
+        if (v === null) {
+          return; // we do not handle the clear-value case, it should not happen, we did not enable it
+        }
```

Comment:
i'm not happy about these sections, but it is either this or using my `unwrap` helper, and this seemed easier to understand. if anyone has some idea how to simplify this problem, just leave comment